UI/Panels/InputConfigPanel.cs

```diff
@@ -116,10 +116,11 @@ private void inputsDataGridView_CellContentClick(object sender, DataGridViewCell
                     } //if
 
                     InputConfigItem cfg;
-                    DataRow row = null;
                     bool create = false;
+                    DataRow row = (inputsDataGridView.Rows[e.RowIndex].DataBoundItem as DataRowView)?.Row;
```

Comment:
I think this should be `var` instead of `DataRow` (style-wise)

Reply:
And elsewhere in this PR as well.

Reply:
I want to keep changes to the  minimum at this point. Thanks for pointing this out though!

Reply:
@neilenns I was thinking about this particular case, and I must say: I cannot easily defer the type of `.Row` and so putting `DataRow` in front of the variable makes that crystal clear.

I also started to use `var` when initializing variables and when it is clear by looking at the right hand side, what data type is used. Makes it shorter and easier to read.

Reply:
Went and looked at what MSFT says about this since I was curious: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions.

> Use [implicit typing](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables) for local variables when the type of the variable is obvious from the right side of the assignment, or when the precise type is not important.

Then they say to use the explicit type if it isn't clear from the right side (as you mention above). I guess the question is whether knowing it is a DataRow is important :D